packages/utils/interactive-hooks/src/Checkbox.hooks.ts

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { IKeyboardEvent } from '@office-iss/react-native-win32';
+
+export type onToggleCallback = (val: boolean) => void;
```

Comment:
tiniest-nit-possible I'm not a fan of shortening 'value' here, as a parameter. We have a number of places we could use [val|value], and more often than not it's value.  if it were internal it wouldn't bug me, regardless it's not an important distinction

Reply:
I think consistency is important, so if value is used more than val, I will change it. Thanks for the heads up!